submissions/xsorter/port-sniffer/program.js

```diff
@@ -0,0 +1,95 @@
+const net = require('net');
+const dns = require('dns');
+const args = require('minimist')(process.argv.slice(2));
+const range = findPortsRange();
+
+const ports = {
+  firstPort: +range[0],
```

Comment:
Don`t need every time add `+` for converting type to number.
Just guarantee, that findPortsRange will return numbers

Reply:
Thanks for advice, changed type in function's return